.github/workflows/images.yml

```diff
@@ -15,6 +15,8 @@ on:
   # NOTE: github.event context is push payload:
   # https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push
   push:
+    branches:
+      - main
```

Comment:
That would take away the ability to update the image in a branch, which we've used a few times, making it easier to handle feature branches without changing the main image.